app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -1087,7 +1088,12 @@ public void onActivityResult(int requestCode, int resultCode, Intent intent)
             } else if (requestCode == Utils.CARD_IMAGE_FROM_FILE_FRONT || requestCode == Utils.CARD_IMAGE_FROM_FILE_BACK) {
                 Bitmap bitmap = null;
                 try {
-                    bitmap = MediaStore.Images.Media.getBitmap(getContentResolver(), intent.getData());
+                    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
```

Comment:
Nitpick, but could you please switch this around, so that it's `if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q)`?

I just prefer to have the new, proper code on top and the fallback in the else, instead of the fallback for old Android versions as the main thing.